File: src/Services/Ordering/Ordering.Application/DTO/CreateOrderDto.cs


namespace Ordering.Application.DTO;

public record CreateOrderDto(
    string UserName,
    decimal TotalPrice,
    string FirstName,
    string LastName,
    string EmailAddress,
    string AddressLine,
    string Country,
    string State,
    string ZipCode, 
    int PaymentMethod,
    string Currency,
    string? CardName);

File: Client/src/app/store/models/Orders/CreateOrderDto.ts
export interface CreateOrderDto {
  userName: string;
  totalPrice: number;
  firstName: string;
  lastName: string;
  emailAddress: string;
  addressLine: string;
  country: string;
  state: string;
  zipCode: string;
  paymentMethod: number; // ← number !!!
  currency: string;
  cardName: string;
}

